docs/content.zh/docs/deployment/cli.md

```diff
@@ -169,6 +169,11 @@ barrier. This will make all registered event-time timers fire, thus flushing out
 is waiting for a specific watermark, e.g. windows. The job will keep running until all sources properly 
 shut down. This allows the job to finish processing all in-flight data.
 
+{{< hint danger >}}
+Use the `--drain` flag if you want to terminate the job permanently.
+If you want to resume the job at a later point in time, then do not drain the pipeline because it could lead to incorrect results.
```

Comment:
Should we clarify that the incorrect results happen when the job is resumed?
```suggestion
If you want to resume the job at a later point in time, then do not drain the pipeline because it could lead to incorrect results when the job is resumed.
```

Reply:
I think this is a good idea. I will update it.